src/main/java/murphy/Murphy.java

```diff
@@ -0,0 +1,161 @@
+package murphy;
+
+import murphy.storage.Storage;
+import murphy.task.Deadline;
+import murphy.task.Event;
+import murphy.task.Task;
+import murphy.task.TaskList;
+import murphy.task.Todo;
+import murphy.ui.Ui;
+
+/**
+ * Main class of the Murphy application, a chatbot which can record and display tasks.
+ */
+public class Murphy {
+    private Ui ui;
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Creates a new Murphy object using the save file at the specified filepath.
+     */
+    public Murphy(String filepath) {
+        ui = new Ui();
+        try {
+            storage = new Storage(filepath);
+            tasks = new TaskList(storage.load());
+        } catch (MurphyException e) {
+            ui.showError(String.format("Error loading save file: %s", e.getMessage()));
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Gets user input continually to add, delete and modify tasks until the user decides to terminate
+     * the program.
+     */
+    public void run() {
+        /*
+            String logo = " ____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println("Hello from\n" + logo);
+        */
+        boolean isClosed = false;
+        while (!isClosed) {
+            String input = ui.getInput();
+            if (input.equals("bye")) {
```

Comment:
Better divide these into another class `public class Parser` in package `Parser`.

Reply:
Yes it's better to create a separate `Parser` class to encapsulate the commands as methods.